style.css

```diff
@@ -0,0 +1,87 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --h1: #172b4d;
+  --paragraph: #344563;
+  --socials: #505f79;
+  --connect: #7f8cff;
+  --navbar: #6070ff;
+}
+
+div {
+  display: flex;
+  width: 70%;
+  margin: 10px auto;
+}
+
+div img {
+  margin-right: 10px;
+  cursor: pointer;
+}
+
+header {
+  display: flex;
+  margin-bottom: 20px;
+}
+
+header div {
+  justify-content: space-between;
+  align-items: center;
+}
+
+h1,
+p,
+.connect,
+span {
+  font-family: "Poppins", sans-serif;
+}
+
+header span {
+  color: var(--navbar);
+  font-weight: 700;
+  font-size: 18px;
+}
+
+section {

```

Comment:
- [ ] It is good practice to use `css classes` instead of HTML tags to target your HTML element. The reasons is, if for example you add another `section` element in the page how would you differentiate it from this one?